addons/mail/models/mail_mail.py

```diff
@@ -40,9 +40,15 @@ def default_get(self, fields):
 
     # content
     mail_message_id = fields.Many2one('mail.message', 'Message', required=True, ondelete='cascade', index=True, auto_join=True)
+
+    mail_message_id_int = fields.Integer(compute='_compute_mail_message_id_int', compute_sudo=True)
```

Comment:
Could you remove lines between fields definition, unless there is a reason to add lines like that ? If you want to order fields, why not a comment per "section or category of fields", but those void lines feel weird to me ...

Reply:
@tde-banana-odoo done :)